### PR TITLE
Enable Java bindings in GDAL RPM

### PR DESCRIPTION
**Pull request by bmarchant**
Enabled Java bindings and updated tabulation, etc.

### CODE REVIEW

**Comment by bmarchant** [comment]
@mattjdnv These changes should fix our problem with the Java bindings upgrade.  I fixed up some of the tabulation in the spec file.

File: src/SPECS/gdal.spec [hunk 1]
```diff
@@ -207,22 +207,42 @@ This package contains development files for GDAL.
...
+%package java
+Summary:	Java modules for the GDAL file format library
+Group:		Development/Libraries
+Requires:	java >= 1:1.8.0
+Requires:	jpackage-utils
+Requires:	%{name}-libs%{?_isa} = %{epoch}:%{version}-%{release}
+
+%description java
+The GDAL Java modules provide support to handle multiple GIS file formats.
+
```
Comment (mattjdnv) [inline src/SPECS/gdal.spec]: Do we need this here?     Provides: %{name}-java-fgdb

File: src/SPECS/gdal.spec [hunk 2]
```diff
@@ -207,22 +207,42 @@ This package contains development files for GDAL.
...
+
+%package javadoc
+Summary:	Javadocs for %{name}
+Group:		Documentation
+Requires:	jpackage-utils
+BuildArch:	noarch
+
```
Comment (mattjdnv) [inline src/SPECS/gdal.spec]: Same for java-doc

Comment (bmarchant) [inline src/SPECS/gdal.spec]: javadoc doesn't have a `Provides:` statement so I don't think so.

File: src/SPECS/gdal.spec (hunk 1 above)
Comment (bmarchant) [inline src/SPECS/gdal.spec]: Nevermind the comment below, I think you are right.  I'm going to add them.

**Comment by mattjdnv** [comment]
I just tweaked the Jenkins job to get rid of the error.

please build rpms

**Pull request merged**